mainnet1/genesis.json

```diff
@@ -363,7 +363,15 @@
       "spam.protection.max.proposals": "3",
       "spam.protection.proposal.min.tokens": "200000000000000000000000",
       "spam.protection.max.delegations": "360",
-      "spam.protection.delegation.min.tokens": "1"
+      "spam.protection.delegation.min.tokens": "1",
+      "governance.proposal.freeform.minProposerBalance": "1000000000000000000",
+      "governance.proposal.freeform.minVoterBalance": "1000000000000000000",
+      "governance.proposal.freeform.minClose": "48h0m0s",
+      "governance.proposal.freeform.requiredParticipation": "0.00001",
```

Comment:
I'd set this to something like 5% or 10%. The proposals are not binding for the protocol but it would still not be good to have many saying do 'x' that passed but don't have much real support behind them.

Reply:
Addressed